packages/variable-editor/src/components/variables/data/variable-utils.test.ts

```diff
@@ -14,84 +27,82 @@ describe('toVariables', () => {
       deepKeyOne: deepValueOne
       deepKeyTwo: deepValueTwo
 `;
-    expect(toVariables(content)).toEqual([
-      {
-        name: 'string',
-        value: 'value',
-        description: '',
-        metadata: { type: '' },
-        children: []
-      },
-      {
-        name: 'number',
-        value: '42',
-        description: '',
-        metadata: { type: '' },
-        children: []
-      },
-      {
-        name: 'boolean',
-        value: 'true',
-        description: '',
-        metadata: { type: '' },
-        children: []
-      },
-      {
-        name: 'mapping',
-        value: '',
-        description: '',
-        metadata: { type: '' },
-        children: [
-          {
-            name: 'mappingString',
-            value: 'mappingValue',
-            description: '',
-            metadata: { type: '' },
-            children: []
-          },
-          {
-            name: 'mappingNumber',
-            value: '43',
-            description: '',
-            metadata: { type: '' },
-            children: []
-          },
-          {
-            name: 'mappingBoolean',
-            value: 'false',
-            description: '',
-            metadata: { type: '' },
-            children: []
-          },
-          {
-            name: 'mappingMapping',
-            value: '',
-            description: '',
-            metadata: { type: '' },
-            children: [
-              {
-                name: 'deepKeyOne',
-                value: 'deepValueOne',
-                description: '',
-                metadata: { type: '' },
-                children: []
-              },
-              {
-                name: 'deepKeyTwo',
-                value: 'deepValueTwo',
-                description: '',
-                metadata: { type: '' },
-                children: []
-              }
-            ]
-          }
-        ]
-      }
-    ]);
-  });
+  variables = [
```

Comment:
Hard to read this tests with all these test data in it. Can we move them to individual files and import them?